integrationtest/v5migrationhelper/orm/pom.xml

```diff
@@ -20,6 +20,7 @@
     <description>Hibernate Search integration tests for the migration helper for Hibernate ORM</description>
 
     <properties>
+        <test.elasticsearch.run.skip>false</test.elasticsearch.run.skip>
```

Comment:
The migration helper doesn't work with Elasticsearch, so we don't test it against Elasticsearch. This line should not be here.